vendors/circom/prover_main.cc

```diff
@@ -138,24 +139,52 @@ int RealMain(int argc, char** argv) {
       .set_help(
           "Verify the proof. By default verify is disabled. Use this flag "
           "to verify the proof with the public inputs.");
+ parser.AddFlag<base::Flag<size_t>>(&num_runs)
+    .set_short_name("-n")
+    .set_long_name("--num_runs")
```

Comment:
```suggestion
    .set_long_name("--num_runs")
    .set_default_value(1)
```

Reply:
Implemented this change in the current commit